dist/angular-form-messages.js

```diff
@@ -115,55 +122,174 @@ angular.module('angularFormMessages').directive('afField', ["$rootScope", "MESSA
 }]);
 
 angular.module('angularFormMessages')
-  .directive('afMessage', ["MessageService", function (
-    MessageService
+  .directive('afMessageLabel', ["$interpolate", "$log", "$translate", "AfMessageService", function (
```

Comment:
Dit is juist ge-ng-annotated. Overigens vind ik het mooier om annotate pas te doen in de geminifide file. Hier is niet het nodig...